dt-mapping/mapping.php

```diff
@@ -436,6 +442,31 @@ public function modify_location_endpoint( WP_REST_Request $request ) {
             return DT_Mapping_Module_Admin::instance()->process_rest_edits( $params );
         }
 
+        public function delete_sublocation( WP_REST_Request $request ) {
```

Comment:
Missing permission check

Reply:
@prykon thanks for this change!
In general, let me mark the conversations as resolved